### PR TITLE
add form to the blog-app

### PR DESCRIPTION
**Pull request by EyasuTeshome**
## In this PR:

- I add like functionality for the post
- I add comment functionality to the post
- Functionality added to increase the likes and comments counter

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: 'author_id'
   # validation
   validates :name, presence: true
-  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```
Comment (arslanbisharat) [inline app/models/user.rb]: - [ ] Let's remove the commented code. This makes it harder to read the code.

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@
   # root "articles#index"
```
Comment (arslanbisharat) [inline config/routes.rb]: - [ ] Let's remove the commented code. This makes it harder to read the code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @EyasuTeshome, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 2 above)
Comment (EyasuTeshome) [reply config/routes.rb]: @arslanbisharat Thank you for the feedback I will make the changes right away

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @EyasuTeshome 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer
- Great communicatio

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**